src/geometry/util.ts

```diff
@@ -0,0 +1,30 @@
+export const earthRadius = 6371008.8;
+
+/** Convert meters to feet */
+export function mToFt(m: number): number {
+  return m * 3.28084;
+}
+
+/** Convert degrees to Radians */
+export function degToRad(degrees: number): number {
+  return ((degrees % 360) * Math.PI) / 180;
+}
+
+/** Convert radians to degrees */
+export function radToDeg(radians: number): number {
+  return (radians * 180) / Math.PI;
+}
+
+/** Given a latitude and zoom level, determine the max distance each segment can be in meters */
+export function getZoomLevelResolution(latitude: number, zoom: number): number {
+  return (
+    ((Math.cos((latitude * Math.PI) / 180.0) * 2 * Math.PI * 6378137) /
+      (512 * 2 ** zoom)) *
```

Comment:
At multiple occasion you had the tile size as an argument with a default value of 512, I think it would be nice to keep this logic here as well.